kaitlynmooney/project-1-cocktail-app
# Carousel v3.1

❗ Please **DO NOT** open a Pull Request without creating an Issue first. Failure to do so will result in the rejection of the pull request.

## Purpose

Carousel functionality is mostly present, some bugs that need to be fixed (will continue to fix through the night)

Fixes #22 #101 #102 #123 

## Changes

New functions: storageToCarousel, pageInit, resetLibrary, updateCarousel, deleteCocktailsFromStorage

## Steps to Reproduce or Test

The "sample" photo is now the default photo that we are using for drinks that don't return a result from Pexels, so the "sample" carousel on page initialization should show those as the carousel images. Storing the drinks into localStorage from the featured section does not auto-update the carousel, but upon page refresh the carousel will have the drinks available on carousel with their associate pictures + names.

## Optional GIF

N/A
